4.2-federated-framework/README.md

```diff
@@ -29,64 +44,236 @@ As a starting point for this deliverable, a Data Workflow Survey for work packag
 | **Data storage**  | EGA (controlled access), dbGap (controlled access), ENA (open access), GEO (open access)  |
 | **Transfer protocol**   |  http(s) (manual), (s)ftp (manual), Globus (manual), Aspera (manual), htsget (automated)  |
 |  **Access rights**|  Per-user basis (no external access), token based (ELIXIR AAI, Switch AAI)  |
-#### Figure 1: Data workflow
-![Survey Data Transfer](survey-data-transfer.png)
+
+### Figure 1: Data workflow
+
+![Survey Data Transfer](survey-data-transfer.drawio.png)
```

Comment:
It looks like there might be a problem with image paths: they are referenced directly from the `/4.2-federated-framework` folder, but are actually committed in `/docs/4.2-federated-framework`. I suggest moving them to `/4.2-federated-framework` for consistency

Reply:
Moved